e2e/test/E2ETests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net472;net451</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1.18;net472;net451</TargetFrameworks>
```

Comment:
Why is 3.1 insufficient to test?

Reply:
Due to nuget framework precedence rules, the netcoreapp3.1 test project will target the ns2.1 dll, while the netcoreapp2.1.18 test project will target the ns2.0 dll